release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -73,6 +73,12 @@ For more information, see xref:../getting_started/openshift-overview.adoc[Gettin
 [id="ocp-4-11-installation-and-upgrade"]
 === Installation and upgrade
 
+[id="ocp-4-11-aws-serial-console-logs"]
+==== Troubleshooting bootstrap failures during installation on AWS
+{product-title} 4.11 gathers serial console logs from the bootstrap and master hosts on AWS if the cluster fails to bootstrap. This log data is added to the standard bootstrap log bundle.
```

Comment:
Clicking through that link, I see this is a place where we still seem to be subject to the use of "master" due to its use in the code. I also see that page uses "control plane, or master, machines" in many places. Would it work to use that here?

Reply:
Good catch, I'm sure it would be OK to just say "control plane" in the release notes. We only start using master later on because it's a flag in the gather command, and the annotation explains it.